src/css/controls/imports/nextup.less

```diff
@@ -13,11 +13,46 @@
     visibility: hidden;
     width: 100%;
 
-    .jw-flag-small-player &,
     .jw-settings-open &,
     .jw-info-open & {
         display: none;
     }
+
+    .jw-flag-small-player & {
+        padding: 0;
+
+        .jw-nextup-title,
+        .jw-nextup-duration,
+        .jw-nextup-close {
+            display: none;
+        }
+
+        .jw-nextup-tooltip {
+            height: 30px;
+        }
+
+        .jw-nextup-header {
+            font-size: 12px;
+        }
+
+        .jw-nextup-body {
+            justify-content: center;
+            align-items: center;
+            padding: 0.75em 0.3em;
+        }
+
+        .jw-nextup-thumbnail {
+            width: 50%;
+        }
+
+        .jw-nextup {
+            max-width: 55px;
+
+            &.jw-nextup-thumbnail-visible {
+                max-width: 110px;
```

Comment:
Spoke with @aliciahurst and we both agree the text looks more balanced in the container with a max width of 120px.
<img width="381" alt="Screen Shot 2019-04-04 at 4 24 27 PM" src="https://user-images.githubusercontent.com/4603952/55586138-36a16080-56f6-11e9-8473-7a54dc67d8b1.png">

Reply:
Also, increase the max-width w/o the thumbnail to 65px.